src/Apps/Home/__tests__/HomeApp.jest.tsx

```diff
@@ -34,6 +34,9 @@ describe("HomeApp", () => {
         featuredEventsOrderedSet: orderedSet(id: "example") {
           ...HomeApp_featuredEventsOrderedSet
         }
+        heroUnitsConnection(first: 10) {
+          ...HomeApp_heroUnitsConnection
+        }
```

Comment:
Hey @dzucconi I'm getting a type error in this test but I can't figure out what else it wants me to do? Shouldn't the relay generator update the test with the right shape? Is there something I'm missing?

Reply:
Here's a link to the error:

https://app.circleci.com/pipelines/github/artsy/force/48962/workflows/7099df50-d67d-424e-b72a-24965647b1ac/jobs/367169

Reply:
What's the type error?

Reply:
Ah, that's just because it can be nullable. We should probably alter the type signature of setupTestWrapper to assert that it'll be there but for now you can either ts-ignore it or null check for the props. If you search for `Component: props =>` you'll see what I mean.